### PR TITLE
Consensus Mask. Removed unnecessary argument.

### PR DESCRIPTION
**Pull request by Frozen**
NewMask doesn't accept `myKey` anymore.

### CODE REVIEW

**Comment by ONECasey** [comment]
Whats this supposed to be merged to `main`? @Frozen

**Review by adsorptionenthalpy — APPROVED** [review]

**Comment by ONECasey** [comment]
@Frozen. Does this go into `main`?

**Review by ONECasey — CHANGES_REQUESTED** [review]
@Frozen Does this go into `main`?

**Comment by Frozen** [comment]
> @Frozen Does this go into `main`?

Changed to dev

File: crypto/bls/mask.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ type Mask struct {
 // cosigners are disabled by default. If a public key is given it verifies that
 // it is present in the list of keys and sets the corresponding index in the
 // bitmask to 1 (enabled).
```
Comment (MaxMustermann2) [inline crypto/bls/mask.go:77]: Can you update the documentation to remove the content `If a public key is given...` ?

**Review by MaxMustermann2 — COMMENTED** [review]
Small comment but otherwise looks good. I confirmed with Konstantin elsewhere that the parameter is not used meaningfully anywhere at all.

**Review by sophoah — APPROVED** [review]
LGTM

**Pull request merged**